bindings/rust/src/lib.rs

```diff
@@ -665,6 +667,31 @@ macro_rules! sig_variant_impl {
             }
         }
 
+        #[cfg(feature = "serde")]
+        impl Serialize for SecretKey {
+            fn serialize<S: Serializer>(
+                &self,
+                ser: S,
+            ) -> Result<S::Ok, S::Error> {
+                serde_bytes::ByteBuf::from(self.serialize()).serialize(ser)
+            }
+        }
+
+        #[cfg(feature = "serde")]
+        impl<'de> Deserialize<'de> for SecretKey {
+            fn deserialize<D: Deserializer<'de>>(
+                deser: D,
+            ) -> Result<Self, D::Error> {
+                let bs: serde_bytes::ByteBuf = Deserialize::deserialize(deser)?;
```

Comment:
Yeah, `bs` gives kind of bad associations... But if we drop `serde_bytes`, we get estate for better variable name :-) `bytes` would work... [Omitting `serde_bytes` would also allow to drop `serde_actual` thing, right?]

Reply:
Right on all counts. Fixed :)